UniWatch/Controllers/ClassController.cs

```diff
@@ -13,7 +13,7 @@ public class ClassController : Controller
     {
         private IDataAccess _dataAccess;
 
-        public ClassController()
+        public ClassController():this(new DataAccess.DataAccess())
         {
             _dataAccess = new DataAccess.DataAccess();
```

Comment:
Remove this line. The assignment is taken care of in your `public ClassController(IDataAccess dataAccess)` constructor.

[This is a good place to read about chained constructors.](https://msdn.microsoft.com/en-us/library/ms173115.aspx)